docs/UserGuide.md

```diff
@@ -216,17 +217,14 @@ Adds a customer to your application.
 Format: `addcust n/NAME p/PHONE e/EMAIL b/BUDGET [c/CHARACTERISTIC]…​`
 
 * `n/NAME`				         : Name of the customer
-* `p/PHONE_NUMBER`		         : Phone number of the customer
+* `p/PHONE_NUMBER`		         : Phone number of the customer (only 8 digits and starting with 6, 8 or 9)
 * `e/EMAIL`				         : Email of the customer
-* `b/BUDGET`		             : Budget of the customer
+* `b/BUDGET`		             : Budget of the customer (5 to 12 digits and starting with a non-zero digit)
 * `c/CHARACTERISTIC` (optional)  : Characteristics of the property the customer is looking for
 
 Examples:
-* `addcust n/Tim Cook p/91234567 e/cook@apple.com b/2500000 c/bright c/sunny`
-* `addcust n/Phoebe p/87654321 e/pb@gmail.com b/200000 c/bright c/sunny c/white`
-
-You should see the message in the [output box](#2-command-input-and-output-boxes) as below when a customer is successfully added. The new customer will be added to the bottom of your list for easy reference!
-![addcustUi](images/user-guide/addcustUi.png)
+* `addcust n/Tim Cook p/91234567 e/cook@apple.com b/2500000 c/bright c/sunny` adds a customer named Tim cook with all the relevant details
```

Comment:
`...adds a customer named Tim Cook with all the relevant details`
Capitalise `Cook` in `Tim Cook`.